Makefile

```diff
@@ -0,0 +1,55 @@
+#This file stores instructions to install openqaoa in developer mode.
+#Currently can only install all packages together for developer mode.
+
+.PHONY: local-install
+	pip install ./src/openqaoa-core
+	pip install ./src/openqaoa-qiskit
+	pip install ./src/openqaoa-pyquil
+	pip install ./src/openqaoa-braket
+	pip install ./src/openqaoa-azure
+	pip install .
+
+.PHONY: dev-install
```

Comment:
What is the difference between local-install and dev-install?

Reply:
`local-install` installs the git cloned package as is, whereas `dev-install` installs the package as editable